packages/ui/src/components/Chip/Chip.tsx

```diff
@@ -59,7 +59,10 @@ const Chip: React.FC<ChipProps> = ({
         color={color}
         mode={mode}
         role="listitem"
-        data-h2-padding="base(x.25, x.5)"
+        data-h2-margin="base(0, x.25, 0, 0)"
+        data-h2-padding="base(x.5)"
+        data-h2-display="base(flex)"
+        data-h2-align-items="base(center)"
```

Comment:
Was this intentional? The chips look a bit larger than the mockup and spacing is a little inconsistent now.

Reply:
I was playing around with this and must of forgot about it, which led to it getting merged in 😅 Fixed by [f8f2928](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/f8f2928ee1f4371ec3c9964c3cc8937c9c6e054b)